### DISCUSSION TITLE
Profile Number Settings

### DISCUSSION THREAD

**Discussion by getsean** [0perationPrivacy/VoIP#201] [Ideas]
Instead of a drop down (which right now is limited to a certain number of numbers) it would be nice if you could just type in which phone number you want to use for the profile. 

![image](https://github.com/0perationPrivacy/VoIP/assets/77656279/1d5ce38a-86c7-4abe-b39a-ef2318ee2c62)

**Comment by 0perationPrivacy** [comment]
Good idea, but I can see issues with this, typos and errors in formatting, and sending in invalid characters for brute forcing. 
With a dropdown, users cannot make a mistake, all options are valid in realtime as they are retrieved from their own account.
I never tested 100's of numbers, but that issue will be addressed as well. It's probably some default limitation in the retrieval.

**Reply by getsean** [reply]
Great thanks so much for working on it! Haven't been able to use the app since I accidentally deleted my profile a few months back.

**Reply by getsean** [reply]
Any update on when this might get fixed? Will send BTC tip for it to be prioritized. ;)

**Reply by 0perationPrivacy** [reply]
this is implemented in the latest v0.94 (auto-complete with manually typing in the number)
Test it out. It may have also fixed the number limitation you were having. I tested and didn't get any limitaiton, I tried dummy data of 100 numbers, they all displayed.

**Comment by getsean** [comment]
Thanks for working on this. Turns out I have 130 numbers.. (I setup voip systems for a client) and for some reason I still can't see all my numbers. :( The one number I need is a number I purchased first, so not sure if Telnyx has a limit or something? I do like that new search feature on the drop down that is great.

**Reply by 0perationPrivacy** [reply]
Odd. I will test with a 150 numbers and report back.

**Comment by getsean** [comment]
Cool thanks for your help. I spent some time on Telnyx also trying to see if maybe it wasn't showing up because of a feature or something I enabled and still couldn't seem to get it on that list. I wish Telnyx would let you assign different numbers to different API keys..

**Reply by 0perationPrivacy** [reply]
tried 150 numbers, they show up fine. could be a telnyx api issue where it's not pulling all numbers.

**Comment by getsean** [comment]
OK thanks for your help on this, I will work with them and see if I can get it to show up.

**Comment by getsean** [comment]
I have a ticket opened with Telnyx, would you mind directing me to the code that is being used in that section of the website? They are asking me questions I don't know how to answer. I am no coder..

**Comment by getsean** [comment]
This is still an issue. Would just love the option to type in what phone number that I want the app to use.
